app/src/main/res/drawable/ic_picture.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:viewportHeight="489.4"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_picture` appears to be unused
 Unused resources make applications larger and slow down builds.